k6ext/metrics.go

```diff
@@ -9,11 +22,39 @@ type CustomMetrics struct {
 	BrowserFirstContentfulPaint *k6metrics.Metric
 	BrowserFirstMeaningfulPaint *k6metrics.Metric
 	BrowserLoaded               *k6metrics.Metric
+
+	WebVitals map[string]*k6metrics.Metric
 }
 
 // RegisterCustomMetrics creates and registers our custom metrics with the k6
 // VU Registry and returns our internal struct pointer.
 func RegisterCustomMetrics(registry *k6metrics.Registry) *CustomMetrics {
+	wvs := map[string]string{
+		webVitalFID:  "webvital_first_input_delay",
+		webVitalTTFB: "webvital_time_to_first_byte",
+		webVitalLCP:  "webvital_largest_content_paint",
+		webVitalCLS:  "webvital_cumulative_layout_shift",
+		webVitalINP:  "webvital_interaction_to_next_paint",
+		webVitalFCP:  "webvital_first_contentful_paint",
+	}
+	webVitals := make(map[string]*k6metrics.Metric)
+
+	for k, v := range wvs {
+		t := k6metrics.Time
+		if k == webVitalCLS {
+			t = k6metrics.Default
+		}
```

Comment:
Please add a comment here why `webVitalCLS` becomes the default metric.

Reply:
Good point. I've added a comment in 09a40503456f286cc776ef55d69e10a970a6ccbb